### PR TITLE
adds audio testfile (sine wave)

### PR DESCRIPTION
**Pull request by ablwr**
Shouldn't just be video test files up in here!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1347,6 +1347,29 @@
...
+          <p>Generate a test audio file playing a Sine wave.</p>
+          <p><code>ffmpeg -f lavfi -i "sine=frequency=1000:sample_rate=48000:duration=5" -c:a pcm_s16le <i>output_file</i>.wav</code></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-f lavfi</dt><dd>tells ffmpeg to use the libavfilter input virtual device <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">[more]</a></dd>
+            <dt>-i "sine=frequency=1000:sample_rate=48000:duration=5"</dt><dd>Sets the signal to 1000 Hz, sampling at 48kHz, and for 5 seconds</dd>
+            <dt>-c:a pcm_s16le</dt><dd>encodes the audio codec in <code>pcm_s16le</code> (the default encoding for wav files). pcm represents pulse-code moderation format (raw bytes), <code>16</code> means 16 bits per sample, and <code>le</code> means "little endian"</dd>
+            <dt><i>output_file</i>.wav</dt><dd>path, name and extension of the output file</dd>
+          </dl>
+          <p class="link"></p>
```
Comment (retokromer) [inline index.html]: You could mention in a note, that there are many possibilities in `lavfi`, e.g. you can evaluate any mathematical function:
```
ffmpeg -f lavfi -i aevalsrc="sin(440*2*PI*t)" -t 10 la.wav
```

Comment (retokromer) [reply index.html]: Sorry, I guess the note «la» is called «A» in English.

File: index.html [hunk 2]
```diff
@@ -1347,6 +1347,29 @@
...
+      <div class="modal-content">
+        <div class="well">
+          <h3>Sine wave</h3>
+          <p>Generate a test audio file playing a Sine wave.</p>
+          <p><code>ffmpeg -f lavfi -i "sine=frequency=1000:sample_rate=48000:duration=5" -c:a pcm_s16le <i>output_file</i>.wav</code></p>
+          <dl>
+            <dt>ffmpeg</dt><dd>starts the command</dd>
+            <dt>-f lavfi</dt><dd>tells ffmpeg to use the libavfilter input virtual device <a href="http://ffmpeg.org/ffmpeg-devices.html#lavfi" target="_blank">[more]</a></dd>
+            <dt>-i "sine=frequency=1000:sample_rate=48000:duration=5"</dt><dd>Sets the signal to 1000 Hz, sampling at 48kHz, and for 5 seconds</dd>
+            <dt>-c:a pcm_s16le</dt><dd>encodes the audio codec in <code>pcm_s16le</code> (the default encoding for wav files). pcm represents pulse-code moderation format (raw bytes), <code>16</code> means 16 bits per sample, and <code>le</code> means "little endian"</dd>
```
Comment (privatezero) [inline index.html]: Just a nit-pick! There is a slight typo- 'Modulation' not 'moderation'

Comment (privatezero) [reply index.html]: Word. Audiooooooo

Comment (retokromer) [reply index.html]: More nit-picking: I would unify: either 1000 Hz and 48 kHz (my favourite) or 1000Hz and 48kHz.

Comment (ablwr) [reply index.html]: Whooooops late-night typing. Correcting both.

File: index.html (hunk 1 above)
Comment (ablwr) [reply index.html]: Ahh I was trying to keep with the exact pattern we use for the other recipes? Maybe an example of this merits its own recipe!

File: index.html [hunk 3]
```diff
@@ -1347,6 +1347,29 @@
...
   <!-- ends Play VGA SMPTE bars -->
 
+  <!-- Sine wave -->
+  <span data-toggle="modal" data-target="#sine_wave"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Generate a test audio file playing a sine wave">Sine wave</button></span>
+  <div id="sine_wave" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Sine wave</h3>
+          <p>Generate a test audio file playing a Sine wave.</p>
```
Comment (retokromer) [inline index.html]: Does sine wear uppercase in English?

File: index.html (hunk 1 above)
Comment (retokromer) [reply index.html]: If another recipe, then we could do something more crazy and combine sounds together.

Comment (retokromer) [reply index.html]: We could include a recipe around `aevalsrc`.

**Comment by ablwr** [comment]
Nooooo idea, hence my wavering! I think not.

**Comment by retokromer** [comment]
According to my books, without capital.

**Comment by retokromer** [comment]
:1st_place_medal:

**Pull request merged**